### PR TITLE
DOC: update function-level docstrings in instruments

### PR DESCRIPTION
**Pull request by jklenzing**
- removes void returns
- standardized parameter lists (no parens, default at end of long description)
- updates template with new load info for xarray
- removes unused bits of template leftover in some instruments (de2, saber)
- uses general list_files for sport_ivm to cut down on untested code

Merge chain: This branch => #480 => #469

### CODE REVIEW

**Comment by jklenzing** [comment]
Looking through coveralls, the removal of unused imports and default functions has resulted in a slight decrease in coverage.  There's a few more areas where imports could be moved to the top to gain back some coverage.  Let me know if I should tweak more, or if we continue as is.

**Comment by aburrell** [comment]
I don't see any "new" lines in the areas where the coverage has dropped, and it's by 0.01%, so I think it's ok.

File: pysat/instruments/templates/madrigal_pandas.py [hunk 1]
```diff
@@ -166,30 +160,24 @@ def _general_download(date_array, tag='', sat_id='', data_path=None, user=None,
...
     kindat : int
-        Madrigal integer code used to identify data set
-
-    Returns
-    --------
-    Void : (NoneType)
-        Downloads data to disk.
+        Madrigal integer code used to identify data set (default=None)
 
     Notes
```
Comment (aburrell) [inline pysat/instruments/templates/madrigal_pandas.py:182]: Should this be Note, or is that changed elsewhere?

File: pysat/instruments/timed_see.py [hunk 2]
```diff
@@ -84,18 +84,14 @@ def clean(inst):
...
+    inst : pysat.Instrument
         Instrument class object, whose attribute clean_level is used to return
         the desired level of data selectivity.
 
-    Returns
-    --------
-    Void : (NoneType)
-        data in inst is modified in-place.
-
     Notes
```
Comment (aburrell) [inline pysat/instruments/timed_see.py:91]: Another 'Note' area.

File: pysat/instruments/ucar_tiegcm.py [hunk 3]
```diff
@@ -208,32 +204,25 @@ def download(date_array, tag, sat_id, data_path=None, user=None, password=None,
...
 
-    Returns
-    --------
-    Void : (NoneType)
-        Downloads data to disk.
-
-
     """
 
     import warnings
```
Comment (aburrell) [inline pysat/instruments/ucar_tiegcm.py:228]: Should this be here?

File: pysat/instruments/templates/madrigal_pandas.py (hunk 1 above)
Comment (jklenzing) [reply pysat/instruments/templates/madrigal_pandas.py:182]: the diff is just formatted weird. description for `kindat : int` at lin 179

Comment (jklenzing) [reply pysat/instruments/templates/madrigal_pandas.py:182]: oops. see what you mean now.  fixing.

File: pysat/instruments/champ_star.py [hunk 4]
```diff
@@ -202,13 +205,13 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
...
     -----------
-    inst : (pysat.Instrument)
+    inst : pysat.Instrument
         Instrument class object, whose attribute clean_level is used to return
         the desired level of data selectivity.
 
     Returns
     --------
-    Void : (NoneType)
+    Void : NoneType
```
Comment (rstoneback) [inline pysat/instruments/champ_star.py:214]: Should the whole returns be removed?

Comment (jklenzing) [reply pysat/instruments/champ_star.py:214]: Looks like I missed this. On the road now, but feel free to update here or downstream

**Comment by rstoneback** [comment]
No worries, please!

Good to know I won’t be stepping on any toes. I’ll change that little bit, let tests run through, and bring everything on in at develop.

I’ll make the pull from develop into master, let the tests run, and ping when it is ready for ‘final review’.

Cheers!
Russell

**Pull request merged**